frontend-react/src/components/FileHandlers/FileHandlerMessaging.tsx

```diff
@@ -92,35 +92,27 @@ export const FileSuccessDisplay = ({
     );
 };
 
-type FileErrorDisplayProps = {
-    errors: FileResponseError[];
-    message: string;
-    fileName: string;
-    handlerType: string;
-    heading: string;
-};
-
 /***
  * This function attempts to truncate an error message if it contains
  * a full stack trace
- * @param truncateErrorMessage - the error message to potentially reformat
+ * @param errorMessage - the error message to potentially reformat
  * @returns - the original or transformed error message
  */
-const reformat = (truncateErrorMessage: string | undefined): string => {
-    if (!truncateErrorMessage) return "";
-
-    if (
-        truncateErrorMessage.includes("\n") &&
-        truncateErrorMessage.includes("Exception:")
-    )
-        return (
-            truncateErrorMessage.substring(
-                0,
-                truncateErrorMessage.indexOf("\n")
-            ) + " ..."
-        );
-
-    return truncateErrorMessage;
+const truncateErrorMesssage = (errorMessage: string | undefined): string => {
```

Comment:
just renaming things

Reply:
oh _that's_ what you meant! yeah, fire me...